commands/slash/clear.js

```diff
@@ -34,7 +34,7 @@ const command = new SlashCommand()
     if (!player.queue || !player.queue.length || player.queue.length === 0) {
       let cembed = new MessageEmbed()
         .setColor(client.config.embedColor)
-        .setDescription("❌ | **Invalid, Not enough track to be cleared.**");
+        .setDescription("❌ | **Invalid arguments, not enough tracks to be cleared.**");
```

Comment:
`clear` doesn't require argument to begin with, why not "Queue is already empty" or "No track to clear"?

Reply:
I don't want to completely change the whole text I just want to fix it up a bit

Reply:
"No track to clear" just sounds so wrong and very basic, I'll make changes to it though

Reply:
> why not "Queue is already empty" or "No track to clear"?

Why not "There's not enough track in the queue to be cleared." and just ditch the whole "Invalid Argument" thing

> "No track to clear" just sounds so wrong and very basic

K.I.S.S.

Reply:
smh alr

Reply:
> "No track to clear" just sounds so wrong and very basic, I'll make changes to it though

Wdym wrong? So you're saying "No track in the queue" or "Queue has no track" and the variation is wrong? And what's wrong with simple and basic? Do you want it to be "In order to clear the current queue in this guild, you will have to run the play command so the queue in this guild contains a track that's added from youtube that's the result of you invoking the play command so the bot can clear the track from the queue that has a track that was added by you through the play command. Unless there's a track in the current queue in this guild the bot can't clear the queue in this guild because in order to clear the current queue in this guild you will have to run the play command so the queue in this guild contains a track that's added from youtube that's the result of you invoking the play command so the bot can clear the track from the queue that has a track that was added by you through the play command. Unless there's a track in the current queue in this guild the bot can't clear the queue in this guild because in order to clear the current queue in this guild you will have to run the play command so the queue in this guild contains a track that's added from youtube that's the result of you invoking the play command so the bot can clear the track from the queue that has a track that was added by you through the play command. Unless there's a track in the current queue in this guild the bot can't clear the queue in this guild because in order to clear the current queue in this guild you will have to run the play command so the queue in this guild contains a track that's added from youtube that's the result of you invoking the play command so the bot can clear the track from the queue that has a track that was added by you through the play command. Unless there's a track in the current queue in this guild the bot can't clear the queue in this guild because in order to clear the ... (69420 trillion more characters)"? Did you invent english? 

> Why not "There's not enough track in the queue to be cleared." and just ditch the whole "Invalid Argument" thing

"not enough" is pretty dumb. "You have not enough track to clear the queue, you need at least 69 track to clear it".

Reply:
Woah chill out you're starting to become like an Indian mom assigning chores